packages/react/toggle-group/src/ToggleGroup.tsx

```diff
@@ -276,4 +276,10 @@ export {
   Root,
   Item,
 };
-export type { ToggleGroupWithSingleProps, ToggleGroupWithMultipleProps, ToggleGroupItemProps };
+export type {
+  ToggleGroupSingleProps,
+  ToggleGroupMultipleProps,
+  ToggleGroupWithSingleProps,
+  ToggleGroupWithMultipleProps,
```

Comment:
Should we get rid of these two then if we're always going to expose the props without the `type`?

Reply:
I removed them everywhere but then the build failed in `ToolbarToggleGroup` because it was looking for those.